src/main/java/pl/edukacyjni/user/UserService.java

```diff
@@ -9,15 +9,38 @@
 @Service
 public class UserService {
 
-    private static final List<User> USERS = new ArrayList<>(){{
+    private static final List<User> USERS = new ArrayList<>() {{
         add(new User(1L, "michaelPaello", "kochamPierogi"));
         add(new User(2L, "theGhostCasper", "huliganJezusaSt0"));
         add(new User(3L, "stNicolas", "haslo123!"));
         add(new User(4L, "santiago", "deadline"));
         add(new User(5L, "was-far", "abcd"));
     }};
 
-    public List<User> getAllUsers() {
-        return USERS;
+    private UserDTO convertUserToUserDTO(User user) {
+        UserDTO userDTO = new UserDTO();
+
+        userDTO.setId(user.getId());
+        userDTO.setUsername(user.getUsername());
+
+        return userDTO;
+    }
+
+    public List<UserDTO> getAllUsers() {
+        ArrayList<UserDTO> USERSDTO = new ArrayList<UserDTO>();
+
+        for (User user : USERS)
```

Comment:
It's good practice to use {} even for oneliners.
Also we can consider using streams here but that approach is also ok.